files/models.py

```
@@ -2271,3 +2277,16 @@ def record_featured_from_frontend(sender, instance, created, **kwargs):
                 source="frontend",
                 created_by=getattr(instance, "_current_user", None),
             )
+
+
+@receiver(post_save, sender=FeaturedVideo)
+def sync_media_featured_fields(sender, instance, created, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `sender`

(ARG001)